fetchcode/api.py

```diff
@@ -26,29 +27,45 @@ class Response:
 - `content_type`: content type of the file
 - `size`: size of the retrieved content in bytes
 - `url`: fetched URL
+- `scheme` : scheme of the URL
```

Comment:
Why return the scheme too?

Reply:
What I thought thay since earlier our code was only for http based URLs, since we are also supporting ftp based URLs, so I should also save the scheme, so what's your thoughts over it? Should we save it or not :)

Reply:
It is already in the URL, so what will you do with the returned value?

Reply:
Okay got this, making changes for this :D